playwright/pages/epochs.page.ts

```diff
@@ -0,0 +1,267 @@
+import { Page, expect } from "@playwright/test";
+import moment from "moment";
+
```

Comment:
Why did you don't put this into the existing epoch-dashboard.page.ts and epoch-detail.page.ts?